test/conftest.py

```diff
@@ -9,3 +11,10 @@ def pytest_runtest_setup(item):
     if 'JAX_ENABLE_x64' in os.environ:
         config.update('jax_enable_x64', True)
     set_rng_seed(0)
+
+
+def pytest_runtest_call(item):
```

Comment:
I would suggest limiting this hook's scope by creating a directory structure
```
numpyro
+-- test/
    +-- conftest.py  # unmodified
    +-- pyroapi/
        +-- conftest.py  # xfail if not implemented only applies to this subdir
        +-- test_pyroapi.py
```
I have used this pattern in funsor and am refactoring pyroapi to use this pattern in https://github.com/pyro-ppl/pyro-api/pull/5

Reply:
Yes, I was a little worried about it affecting other tests. This sounds like a good way to handle it.